data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,338 @@
+			choice
+				`	Explain the situation.`
+					goto explanation
+				`	"I need advice regarding the species in this region."`
+			`	It looks at you curiously, "So you met them? I am surprised that you were able to communicate, it took a very long time even for us to decipher their way of communication."`
+			`	You quickly explain what happened and the Quarg seens very relieved about the fact that you were not able to talk to them, it probably was a big achievement for them.`
+			label explanation
+			`	"A most peculiar situation you ran into, should not be able to cross the borders of their system yet this species..." It tilts it head for a moment considering what you just said. "To help them I would love, but no ships we have that on gas giants can land and too early it is for us to reveal ourselves to them, I fear nothing there is we can do." It shakes its head.`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"How can you make such a cold hearted decision?"`
+			`	"This difficult for you to understand might be, but the greater good over the sole life I have to place. If ourselves show to them now by the shock of such and advanced species existing will be created chaos."`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"You can deal with the consequences later, help them now!"`
+			`	"Insisting you sure are human, understand I can your point but intervention by us out of question it is. But a proposal for you I have, to you, a solo captain the means to help I will grant, of course some conditions there are."`
+			choice
+				`	"Sure let me do the work."`
+				`	"If you are too weak..."`
+			`	Something like a giggle comes from the Quarg, before continuing.`
+			label initiative
+			`	"Your initiative most welcome is human, but not the tools for such a mission you have. To my knowledge no vessels capable of landing on gas giants ever developed by your kind. And as said you, not capable of communicating with them you are."`
+			`	"But help you I can at one point, communication that is. I can provide you with a single one of our translators, which will allow you to contact the species in their homesystem. To sell you one of their ships ask to them, easy it wont be."`
+			`	It pauses before getting very serious, "but remember one thing you have to, do not to give them any advanced technology of yours. If you agree to that point help you we shall."`
+			choice
+				`	"If that is how I can help them."`
+					goto yes
+				`	"Yes, but why do you want me not to sell my tech?"`
+			`	"Interrupting the natural grow of a species not predictable the consequences are even letting you contact them a great risk is. But that one willing to take I am for the lives of these souls."`
+			label yes
+			`	"Happy I am that you agree to that condition. Please for a moment wait here," it says and hurrilly walks away.`
```

Comment:
```suggestion
			`	"Happy I am that you agree to that condition. Please for a moment wait here," it says and hurriedly walks away.`
```
Maybe `quickly` or `in a hurried manner` or leave it out altogether.

Reply:
Leaving it out seems like the best option

Reply:
```suggestion
			`	"Happy I am that you agree to that condition. Please for a moment wait here," it says and walks away.`
```